### PR TITLE
AMP-27971: Remove Mondrian ETL

### PR DESCRIPTION
**Pull request by developster**

### CODE REVIEW

File: amp/WEB-INF/src/org/dgfoundation/amp/reports/converters/AmpReportFiltersConverter.java [hunk 1]
```diff
@@ -162,7 +162,7 @@ public AmpARFilter buildFilters() {
     /**
      * Convert one of the Mondrian Filters to one of the fields in AmpARFilters and add it to the new filter.
```
Comment (jdeanquin-dg) [inline amp/WEB-INF/src/org/dgfoundation/amp/reports/converters/AmpReportFiltersConverter.java]: Please remove Mondrian from comment

**Comment by jdeanquin-dg** [comment]
Are this views still needed?
v_etl_diagnostics
v_etl_log_activities
v_etl_log_translations

Also I saw you renamed Mondrian named variables we still have one here
AmpReportFiltersConverter#addDateRangeFilter
also report name in 
AmpSchemaPledgesTests#testActualPledgeDetailDatesAndContacts might be renamed to avoid confusions since mondrianhas been removed

**Comment by jdeanquin-dg** [comment]
also please fix conflicts so it can be tested and merged on Monday

**Comment by developster** [comment]
> Are this views still needed?
> v_etl_diagnostics
> v_etl_log_activities
> v_etl_log_translations

These are developer views for easier inspection of amp_etl_changelog table. They might be useful sometime.

**Pull request merged**